devfiles/scala-sbt/meta.yaml

```diff
@@ -3,3 +3,5 @@ displayName: Scala
 description: Scala Stack with OpenJDK 11 and sbt 1.x
 tags: ["Scala", "OpenJDK", "sbt", "Debian"]
 icon: /images/scala.svg
+links:
+  v2: https://github.com/sideeffffect/console-scala-simple/tree/patch-1
```

Comment:
```suggestion
  v2: https://github.com/che-samples/scala-sbt/tree/devfilev2
```

Reply:
@sideeffffect let's update the link to use devfile form the che-samples

Reply:
Thanks